jobs/rpmbuild.yaml

```diff
@@ -18,10 +18,10 @@
             EOF
             cat task.env
             rsync -Hrlptv --stats -e ssh task.env rsync-password.sh builder@${{DUFFY_HOST}}:${{JOB_NAME}}
-            rsync -Hrlptv --stats --delete -e ssh aos-ci/ builder@${{DUFFY_HOST}}:${{JOB_NAME}}/aos-ci
+            rsync -Hrlptv --stats --delete -e ssh aos-ci/ builder@${{DUFFY_HOST}}:${{JOB_NAME}}/ci-pipeline
```

Comment:
Pretty sure the aos-ci in this line needs to be changed to ci-pipeline as well (right after the ssh)